awswrangler/data_api/rds.py

```diff
@@ -0,0 +1,151 @@
+"""RDS Data API Connector."""
+from __future__ import annotations
```

Comment:
This appears to be available [only starting from 3.7](https://www.python.org/dev/peps/pep-0563/#enabling-the-future-behavior-in-python-3-7). 

Unit/integration tests currently run against 3.7 only so wouldn't catch that but we still try to keep 3.6 supported and test every release against 3.6, 3.7, 3.8 & 3.9. Can we remove it and replace string type hints where applicable?

Reply:
Good catch, yes I'll fix that. Looks like the latest test failures are due to the code build job needing new IAM permissions for the Data API calls (i.e. ExecuteStatement, etc.)

Reply:
Updated now. I was able to remove the dependency by just moving the functions to the end of the files below the class definition.

As those functions are the main interface I thought it would be helpful for them to be at the top of the file rather than buried at the end, but it was just a "nice to have". Could probably keep them at the top and use string hints, but moving them to the end of the file seemed like the simpler/cleaner solution.

Reply:
Thanks! Yes, I agree it's cleaner. I added `"rds-data:ExecuteStatement" & "redshift-data:ExecuteStatement"` & re-triggered the tests - let's see if there is anything else it needs.